### ISSUE TITLE
Seems than the serbian tv series of your torrents are dead

### ISSUE THREAD

**Issue by VukDabizljevic** [btdig/dhtcrawler2#19]
Hi, i m interested in this magnet:?xt=urn:btih:7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f&dn=Porodicno+blago+ep+1-62+by+djukahr&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.opentrackr.org:1337/announce your torrent but seems like is totally dead,how to get it?Or second one one this same topic -> https://btdig.com/7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f/porodicno-blago
Vuk Dabizljevic
Thank you.

**Comment by ghost** [comment]
I dont think you understand how this works..  its not 'their torrent'. This application just scans for magnet files on DHT, and allows you to search what was discovered. That is it. Nothing more nothing less. The creators of this application dont 'own' or 'control' any of the content it may reference.

**Comment by VukDabizljevic** [comment]
https://btdig.com/0c51759b78b214688d01528e610e7bb8a9c145af/srecni-ljudi fix them!

**Comment by ghost** [comment]
Go get educated.

**Comment by BiatuAutMiahn** [comment]
> magnet:?xt=urn:btih:7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f&dn=Porodicno+blago+ep+1-62+by+djukahr&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.opentrackr.org:1337/announce`

This is called a magnet link.

The `7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f` in that link represents the fingerprint of the torrent and all of it's data. Just like your fingerprint does not describe your physical appearance and/or your personality, this magnet link cannot describe the files that need to be downloaded, nor the information to verify if they have been downloaded correctly.

The `&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.opentrackr.org:1337/announce` section describes the trackers. Trackers maintain a list of hashes, and who is downloading/uploading the torrent's data.

When your BitTorrent program opens the magnet link, it sends the hash to the trackers in 2 ways. 1, an announcment saying "Hey I have this hash!" or 2, a scrape. where we ask the tracker if they know of any peers for the given hash.

In the event that the torrent has no peers/seeds/leechers then there is literally nobody to share the information for the torrent data itself let alone the files. Even if you happen to have the full .torrent file, if there is nobody that has that data, then it simply does not exist.

So unless you happen to find trackers that have seeds/peers will not get that show.

**Comment by BiatuAutMiahn** [comment]
You will have more luck looking for different torrents that have similar content, archive.org, google around for links to actively hosted files, etc.

**Comment by CSJ-2024** [comment]
> > magnet:?xt=urn:btih:7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f&dn=Porodicno+blago+ep+1-62+by+djukahr&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.opentrackr.org:1337/announce`
> 
> This is called a magnet link.
> 
> The `7ce7e4fa0b43d5930e21612ccdee50a4eb770f5f` in that link represents the fingerprint of the torrent and all of it's data. Just like your fingerprint does not describe your physical appearance and/or your personality, this magnet link cannot describe the files that need to be downloaded, nor the information to verify if they have been downloaded correctly.
> 
> The `&tr=udp://tracker.openbittorrent.com:80&tr=udp://tracker.opentrackr.org:1337/announce` section describes the trackers. Trackers maintain a list of hashes, and who is downloading/uploading the torrent's data.
> 
> When your BitTorrent program opens the magnet link, it sends the hash to the trackers in 2 ways. 1, an announcment saying "Hey I have this hash!" or 2, a scrape. where we ask the tracker if they know of any peers for the given hash.
> 
> In the event that the torrent has no peers/seeds/leechers then there is literally nobody to share the information for the torrent data itself let alone the files. Even if you happen to have the full .torrent file, if there is nobody that has that data, then it simply does not exist.
> 
> So unless you happen to find trackers that have seeds/peers will not get that show.

That was a good explanation. I created a torrent using qBittorrent but that torrent doesn't appear here in bTDig, do you have any advice?

**Comment by BiatuAutMiahn** [comment]
btdig watches the DHT traffic for torrent info. There's no telling if and when it'll pick it up.

**Comment by CSJ-2024** [comment]
> btdig watches the DHT traffic for torrent info. There's no telling if and when it'll pick it up.

Solved!
I just had to upload my torrent to "extranet torrentbay" e people started to download it, and now it's on  BTDigg. It's just perfect.
